src/main/java/uk/gov/hmcts/juror/api/moj/repository/AppearanceRepository.java

```diff
@@ -75,7 +75,11 @@ List<Appearance> findAllByCourtLocationLocCodeAndJurorNumberAndAttendanceDateInO
     @Query(value = "select nextval('juror_mod.attendance_audit_seq')", nativeQuery = true)
     Long getNextAttendanceAuditNumber();
 
-    long countByJurorNumberAndAppearanceStageNotNull(String jurorNumber);
+    @Query(value = "select count(*) from Appearance a where a.jurorNumber= ?1 "
+        + "and a.appearanceStage is not null "
+        + "and( a.attendanceType is null or a.attendanceType not in (uk.gov.hmcts.juror.api.moj.enumeration.AttendanceType.ABSENT))")
```

Comment:
do we need a fully qualified name for ABSENT here?

Reply:
Liniting complained but tested without being fully qualified and all worked as expected... Code updated :)